allensdk/brain_observatory/vbn_2022/metadata_writer/lims_queries.py

```diff
@@ -607,12 +609,53 @@ def _ecephys_structure_acronyms_from_ecephys_session_id_list(
     return struct_tbl
 
 
+def _filter_on_death_date(
+        behavior_session_df: pd.DataFrame,
+        lims_connection: PostgresQueryMixin) -> pd.DataFrame:
+    """
+    Given a pandas dataframe of behavior sessions, remove those
```

Comment:
For future us, could you also add a small note as to "This is due to a user-error where the incorrect session id was used" or something to that effect.

Reply:
I added it in the `Notes` section of the docstring. I probably used more words than you intended.